js/utils/whitelistHandler.js

```diff
@@ -35,7 +52,7 @@ function isWhitelisted(url) {
  */
 function checkWhitelist(url, whitelist) {
     let whitelistArray = whitelist.split(',');
-    const parsedUrl = parseDomain(url);
+    const parsedUrl = parseDomain(url, {customTlds});
```

Comment:
seems like having to add this customTlds is not a great way to go - just yet another config item for customer to have to think about and possibly cause confusion.  and also seems like it is just because this lib 'parseDomain' being used is not that great.  seems like writing our code might be better way to go, likely even simple regex could be used.

Reply:
makes sense, will remove the dependency and write our own code. but, validating the tlds will have to be removed because we'll have to maintain a list of them to validate against which would come back to the same point. in any case, since we are validating against the whitelist array, should be fine even in case of a harmful url.

Reply:
addressed in https://github.com/symphonyoss/SymphonyElectron/pull/350